### ISSUE TITLE
Color Contrast Violation When Text is Inside an Element with a Background Image

### ISSUE THREAD

**Issue by Cassidie** [dequelabs/axe-core#1650]
Expectation: Text color should not fail color contrast test when it's in an element with a background image.

Actual: We have an h1 tag in a div that has a background image. Axe is comparing the body tag's background color to the h1 text color, which is indeed poor color contrast, but our image is dark enough to see the text.

Motivation: Please consider ignoring text inside an element that has a background image set.

*This is kinda related to https://github.com/dequelabs/axe-core/issues/560, but this is regarding it flagging with background-image, not color.

<pre><code>
axe-core version: 3.8.0
axe-webdriver, extension or other integration version: chrome (v74) dev-tools extension

Browser and Assistive Technology versions

For Tooling issues:
- Node version: 10.15.3
- Platform:   Windows 
</code></pre>

**Comment by straker** [comment]
Thanks for letting us know about this. Would you have some time to send me an example where this happens?

**Comment by dylanb** [comment]
@Cassidie did you check to see whether the issue is being raised as a "Needs Review" issue?

**Comment by Cassidie** [comment]
Sorry for not providing an example. We're testing the landing page of `www.estatesales.net`. If you run it on the landing page, you'll see the first violation of "Elements must have sufficient color contrast" is an h1 nested inside 3 divs. There is a sibling div that is absolutely positioned with a background image. 

This is our first time really exploring axe, b/c since the update our lighthouse scores have gone way down. We're not sure how to interpret some of these violations that don't seem like violations in our case. We appreciate any help!

**Comment by Cassidie** [comment]
> @Cassidie did you check to see whether the issue is being raised as a "Needs Review" issue?

I'm not sure what you're referring to. I didn't see a label on the other case. How does one raise an issue as Needs Review?

**Comment by straker** [comment]
Thanks for the link. I believe the problem is that the background-image is added as a `::before` pseudo element, which isn't part of the `elementsFromPoint` API that we use to determine which elements are behind the text. We have a [similar issue for pseudo elements and linear-gradients](https://github.com/dequelabs/axe-core/issues/975) that we are still looking into how to support.

**Comment by Cassidie** [comment]
Oh great! Thank you for your response! I saw that case while I was reviewing current issues, but I didn't even think about that being the reason. Thank you very much for the info, and I'll close this case.
